infor.text

```diff
@@ -0,0 +1,19 @@
+Y
+2
+Kien-Dai La Hanoi
```

Comment:
Cậu chỉnh lại code để đọc toàn dấu xuống dòng hoặc toàn dấu -, hoặc là có quy tắc lúc này đọc dấu , lúc kia đọc dấu -. Chứ đừng đọc , với - mà không có quy tắc, như thế sẽ khó bảo trì code (tức khó đọc lại tìm lỗi) nhé